src/Components/FileExplorer/Component_FileExplorer.re

```diff
@@ -83,7 +83,10 @@ module Effects = {
 type outmsg =
   | Nothing
   | Effect(Isolinear.Effect.t(msg))
-  | OpenFile(string)
+  | OpenFile({
+      filePath: string,
+      preview: bool,
+    })
```

Comment:
Would it be better to have `OpenFile(str)` and `PreviewFile(str)`, instead of `OpenFile(str, bool)`? (I don't know the issues that would cause, but making it explicit that it is a preview could be nice?)